src/NuGet.Clients/PackageManagement.VisualStudio/Strings.resx

```diff
@@ -236,4 +236,12 @@
   <data name="CredentialProviderFailed_ImportedProvider" xml:space="preserve">
     <value>Failed to load credential provider from assembly {0}.</value>
   </data>
-</root>
+  <data name="Error_MultipleFrameworksUwp" xml:space="preserve">
+    <value>Failed to migrate project '{0}' as there are multiple target frameworks in project.json file.</value>
```

Comment:
suggestion: Failed to migrate project '{0}'.  UAP projects may only contain one target framework.